cost/azure/idle_compute_instances/azure_idle_compute_instances.pt

```diff
@@ -376,6 +376,9 @@ parameters "param_cpu_average_percentage","ds_azure_instance_performance"
 result "results"
 code <<-EOS
   var results = []
+  function roundNumber(num, dec) {
```

Comment:
This isn't necessary. You can use the built-in toFixed() method to round a number, and you can use parseFloat() to convert it from a string back to a float if necessary.

Here's an example from the node console:

> number = 1.23456789
1.23456789
> number.toFixed(2)
'1.23'
> parseFloat(number.toFixed(2))
1.23